mbtools/patch_pset_retry_limits.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+from pathlib import Path
+from mbtools import utils
+from bs4 import BeautifulSoup
+from mbtools.generate_mbz_toc import parse_toc
+import os
+import json
+
+
+def get_lesson_pages(mbz_path):
```

Comment:
Minor: Can we name this `get_lesson_practice_pages`? It actually confused me a bit when reading the code, and it may help future readers as well.